distribution/docs/src/main/resources/_contents/_tables/Csw_Connected_Source-table-contents.adoc

```diff
@@ -0,0 +1,139 @@
+.[[Csw_Connected_Source]]CSW Connected Source
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Source ID
+|id
+|String
+|The unique name of the Source
+|CSW
+|true
+
+| CSW URL
+| cswUrl
+| String
+| URL to the endpoint implementing the Catalogue Service for Web (CSW) spec
+| null
+| true
+
+| Event Service Address
+| eventServiceAddress
+| String
+| DDF Event Service endpoint. Do NOT include .wsdl or ?wsdl.
+| null
+| false
+
+| Register for Events
+| registerForEvents
+| Boolean
+| Check to register for events from this connected source.
+| false
+| false
+
+| Username
+| username
+| String
+| Username for CSW Service (optional)
+| null
+| false
+
+| Password
+| password
+| String
+| Password for CSW Service (optional)
+| null
+| false
+
+| Disable CN Check
+| disableCnCheck
+| Boolean
+| Disable CN check for the server certificate. This should only be used when testing.
+| false
+| true
+
+| Force Longitude/Latitude coordinate order
+| isLonLatOrder
+| Boolean
+| Force Longitude/Latitude coordinate order
+| false
+| true
+
+| Use posList in LinearRing
+| usePosList
+| Boolean
+| Use a <posList> element rather than a series of <pos> elements when issuing geospatial queries containing a LinearRing
```

Comment:
Yes. these tables were scripted from the metatype.xml files and they are inconsistent there too. Updated.